server/sql_app/main.py

```diff
@@ -580,23 +613,47 @@ def parcel_fill(parcel_fill: schemas.ParcelFill,
     return job_db
 
 
-@app.post("/stats_under_parcel/", response_model=schemas.JobResponse)
-def get_lulc_stats_under_parcel(parcel_stats_req: schemas.ParcelStatsRequest,
-                                db: Session = Depends(get_db)):
-    # TODO: Check if this parcel has already been computed.
-    # NOTE this assumes we're always using baseline LULC
+@app.post("/remove_parcel/")
+def remove_parcel(delete_parcel_request: schemas.ParcelDeleteRequest,
+                  db: Session = Depends(get_db)):
+    status = crud.delete_parcel(db=db, **delete_parcel_request.dict())
```

Comment:
That sounds smart. I followed your example `crud.delete_scenario` to return a `404` if the parcel does not exist in the DB. But I don't think the frontend actually cares about the status of this request. If the delete failed because the parcel doesn't exist, mission accomplished from the frontend perspective.